packages/preview-middleware/src/base/flp.ts

```diff
@@ -142,11 +142,12 @@ export class FlpSandbox {
         const supportedThemes: string[] = (manifest['sap.ui5']?.supportedThemes as []) ?? [DEFAULT_THEME];
         const ui5Theme =
             this.config.theme ?? (supportedThemes.includes(DEFAULT_THEME) ? DEFAULT_THEME : supportedThemes[0]);
+        const depsLibs = this.rta ? ['sap.ui.rta'] : [];
         this.templateConfig = {
             basePath: relative(dirname(this.config.path), '/') ?? '.',
             apps: {},
             ui5: {
-                libs: Object.keys(manifest['sap.ui5']?.dependencies?.libs ?? {}).join(','),
+                libs: [...Object.keys(manifest['sap.ui5']?.dependencies?.libs ?? {}), ...depsLibs].join(','),
```

Comment:
Please remove, I added the missing `sap.ui.rta` with https://github.com/SAP/open-ux-tools/pull/1363 and it is added only for the cases when you want to start Runtime Adaptation.

Reply:
removed

Reply:
thanks